Detectors/CPV/simulation/src/Detector.cxx

```diff
@@ -351,8 +351,6 @@ void Detector::AddHit(int trackID, int detID, const Point3D<float>& pos, double
   LOG(DEBUG4) << "Adding hit for track " << trackID << " in a pad " << detID << " with position (" << pos.X() << ", "
               << pos.Y() << ", " << pos.Z() << "), time" << time << ", qdep =" << qdep << std::endl;
   mHits->emplace_back(trackID, detID, pos, time, qdep);
-  // register hit creation with MCStack
-  static_cast<o2::data::Stack*>(fMC->GetStack())->addHit(GetDetId());
```

Comment:
this has to stay. it is required to notify someone that a track left a hit in CPV.

Reply:
Sandro,  David, I do not plan to touch CPV yet  - I will return this lost line. 
Presently I implemented filling of MCLabels for PHOS, but I did not try it with DPL framework yet. With old-fashined approach with DigitiserTask I still fighting with problem 
[INFO] CREATING BRANCH PHOSDigitMCTruth
[FATAL] No dictionary found for o2::dataformats::MCTruthContainer<o2::phos::MCLabel>
Probably it is better to postpone merging till I will manage to run digitization with DPL. 
Or it is easier to merge now as is and continue with new clang formatting? What do you suggest?  Thanks, Dmitri